### PR TITLE
Add wrapper to T3 for CommonJS

### LINKED ISSUE #24: Load via npm as module
Now, there is no way to load ```t3js``` properly via browserify or webpack, as package.json does not have any main file entry, so I can't just write:
```javascript
import Box from "t3js";
```

But it would be very nice to have such opportunity (not to use global Box variable, but the one exported by module).

### PR DESCRIPTION
**Pull request by j3tan**
- wrapping t3 with a module.exports shim

fixes #24

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @j3tan, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: lib/exports/wrap-end.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+	// CommonJS/npm, we want to export Box instead of assigning to global Window
```
Comment (nzakas) [inline lib/exports/wrap-end.js]: Since this and wrap-start are invalid js files, it might be better to have them issue of lib/ with a different extension.

Comment (j3tan) [inline lib/exports/wrap-end.js]: do you mean inside of? I'm going to move them there with a .partial extension

Comment (nzakas) [inline lib/exports/wrap-end.js]: Yuck auto complete. I meant outside. Another option would be to have wrapper.js with a comment as a placeholder for where the code should end up.

**Comment by j3tan** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @j3tan has just signed the CLA. Thanks, and we look forward to your contribution.

**Comment by nzakas** [comment]
+2

**Pull request merged**